src/main/java/ludogorie_soft/reservations_platform_api/entity/Property.java

```diff
@@ -0,0 +1,38 @@
+package ludogorie_soft.reservations_platform_api.entity;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+@Data
+@Table(name = "properties")
+public class Property {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private UUID id;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User owner;
+
+//    private String name;
+//    private String type;
```

Comment:
Remove commented lines

Reply:
After testing this branch, I found this should be GenerationType.UUID
`@Id
    @GeneratedValue(strategy = GenerationType.UUID)
    private UUID id;`

Reply:
Removed commented out lines from code. Changed GenerationType to UUID.